aws/resource_aws_s3_bucket_notification_test.go

```diff
@@ -477,19 +477,26 @@ resource "aws_sns_topic" "topic" {
 
   policy = <<POLICY
 {
-	"Version":"2012-10-17",
-	"Statement":[{
-		"Sid": "",
-		"Effect": "Allow",
-		"Principal": {"AWS":"*"},
-		"Action": "SNS:Publish",
-		"Resource": "arn:${data.aws_partition.current.partition}:sns:*:*:%[1]s",
-		"Condition":{
-			"ArnLike":{"aws:SourceArn":"${aws_s3_bucket.bucket.arn}"}
-		}
-	}]
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "",
+      "Effect": "Allow",
+      "Principal": {
+        "AWS": "*"
+      },
+      "Action": "SNS:Publish",
+      "Resource": "arn:${data.aws_partition.current.partition}:sns:*:*:%[1]s",
+      "Condition": {
+        "ArnLike": {
+          "aws:SourceArn": "${aws_s3_bucket.bucket.arn}"
+        }
+      }
+    }
+  ]
 }
 POLICY
+
```

Comment:
This extra whitespace can be removed

Reply:
Can we leave this for a future pass? The overwhelming majority of HEREDOC's have an empty line after so I've been doing it that way to be consistent - consistently wrong I guess.